bin/main.ml

```diff
@@ -139,11 +136,11 @@ let test_cmd =
   in
   Cmd.v info term
 
-let cmd =
+let cmd : (unit, string) result Cmd.t =
   let doc = "A tool to list revdeps and test the revdeps locally" in
   let exits = Cmd.Exit.defaults in
-  let term = Term.(ret (const (fun _ -> `Help (`Pager, None)) $ const ())) in
+  (* let term = Term.(ret (const (fun _ -> `Help (`Pager, None)) $ const ())) in *)
```

Comment:
Why leave this in as a comment? I'm okay with dropping the default of showing the help pager, and just showing an error, instead.

Reply:
This was an oversight on my part. I think defaulting to help is great until/unless we find a preferable default behavior.

Thanks for catching this. Fixed in 63e7012c9a469a8feb6bada142a559a0eebd028a